Library/Homebrew/formula.rb

```diff
@@ -2206,9 +2206,16 @@ def method_added(method)
     # @!attribute [w]
     # The SPDX ID of the open-source license that the formula uses.
     # Shows when running `brew info`.
-    #
+    # Multiple licenses means that the software is licensed under multiple licenses.
+    # Do not use multiple licenses if e.g. different parts are under different licenses.
     # <pre>license "BSD-2-Clause"</pre>
-    attr_rw :license
+    def license(args = nil)
+      if args.nil?
+        @licenses
+      else
+        @licenses = Array(args) unless args == ""
```

Comment:
```suggestion
        @licenses = Array(args)
```

Reply:
Hi @MikeMcQuaid, I might stick with `@licenses = Array(args) unless args == ""` because I might not want the DSL `license ""`-> `[""]`, but instead `nil`. What do you think?

Reply:
Hi @mikejs I mike want to stick with `@licenses = Array(args) unless args == ""` as I think it would be clearer if the DSL `license ""` is interpreted as `nil` instead of `[""]`.